models.py

```diff
@@ -134,7 +134,6 @@ def serialize(self):
         return {
             'id': self.id,
             'criterion_id': self.criterion_id,
-            'created_at': self.created_at,
```

Comment:
I don't think the CMS or the map frontend care when a particular score was created. If we use this info for data science purposes, we can revisit this, but for now I think it's okay to leave it off.